server.js

```diff
@@ -108,11 +122,15 @@ app.post('/', function(req, res) {
     }, function(err, configRes) {
       // default config
       var repoConfig = {
-        userBlacklist: []
+        userBlacklist: [],
+        maxReviewers: 3
       };
 
       if (!err && configRes) {
-        try { repoConfig = JSON.parse(configRes); } catch (e) {}
+        try {
+          merge(repoConfig, JSON.parse(configRes));
```

Comment:
Can you enable the object spread transform so that you can write the following:

```javascript
repoConfig = { ...repoConfig, JSON.parse(configRes)) }
```